packages/@orchidui-vue/src/StoreDesign/SDSidebar/SDSidebar.stories.js

```diff
@@ -1,9 +1,11 @@
 import { Theme, Button } from "@/orchidui";
 import { SDSidebar } from "@/orchidui/StoreDesign";
-// config
-import { Config } from "./settings/Config.sample";
-// data
-import { StoreDesign } from "./settings/StoreDesign.sample";
+// config default
+// import { Config } from "./settings/Config.sample";
+// import { StoreDesign } from "./settings/StoreDesign.sample";
```

Comment:
```suggestion
```

Reply:
there are 2 different settings, default theme and juzzy theme. this only for json generator. i need it for test and regenerate json